recipes/fgbio/meta.yaml

```diff
@@ -15,7 +15,7 @@ source:
   url: https://github.com/fulcrumgenomics/fgbio/archive/{{ tag }}.tar.gz
   #fn: fgbio-{{ version }}.jar
   #url: https://github.com/fulcrumgenomics/fgbio/releases/download/{{ version }}/fgbio-{{ version }}.jar
-  md5: c770b50025284f4baf9dceb45a7fd826
+  sha256: e3f1b73fc623741430dfc0058f7835a8cdc6d7fd6272d85a84fd7215d378d0d6
```

Comment:
Since this is a release, can you use fn and url on lines 16-17?

Reply:
Done; please see next commit.